.pipelines/pipeline.user.windows.buddy.yml

```diff
@@ -5,7 +5,7 @@ environment:
     version: 2016
   runtime:
     provider: appcontainer
-    image: 'cdpxwin.azurecr.io/legacy/global/vse2017u5-external-azsdk-ext-win14393.1884:1.1-node10'
+    image: 'cdpxwin1809.azurecr.io/global/vse2019:latest'
```

Comment:
What does changing this image do?

Reply:
it is having the build infra use a different machine image to run the build.  they required everything move to a version of windows on 1809+ for security updates, and then the rest of the line is stuff like what version of VS, etc.  in this case, workbook doesn't really need any of those dependencies since it's all just scripts.